torch/_inductor/codecache.py

```diff
@@ -1133,6 +1136,16 @@ def iterate_over_candidates() -> Generator[CompiledFxGraph, None, None]:
 
             write_atomic(artifact_path, code, make_dirs=True)
 
+        if bundle := graph._triton_bundle:
+            triton_bundler_meta = TritonBundler.write(bundle)
+            if (meta := triton_bundler_meta) is not None:
+                cache_info["triton_bundler_meta"] = str(meta)
+                add_remote_cache_time_saved(
+                    meta.total_compile_time_ns,
+                    is_backward=is_backward,
+                    triton_bundler=True,
+                )
```

Comment:
You're doing this completely symmetrically to the old column but we know there are problems with the old column so I'd like to do this correctly from the get go.

So what exactly do we want out of this field. I want to know if I had a bundled cache hit or not. So that seems like the bare minimum instrumentation to have. Might be useful to know how many kernels / how big the kernels I hit on in the bundle are. Basically, anything you would have logged to the pt2_remote_cache table, log it here instead, with appropriate aggregation in case there are multiple Inductor compiles in the dynamo_compile span.

Now, how about the actual timings? I think I am interested in the timing that I put into the NCCL ephemeral timeout, because I may want to specifically debug problems where this timeout number is too large. I don't think I'm interested in imputed savings here though (which is what the current field naming suggests). There are too many caveats (parallel versus sequential, also our recent findings about DtoH syncs in compiler causing unpredictable interactions with the in flight eager operations) that I really only want to rely on natural A/B test for sizing the impact here to start.

So concretely, don't call this remote cache time saved. Call it something like nccl ephemeral timeout added or something.